ui/package.json

```diff
@@ -2,11 +2,54 @@
   "name": "kafka-java-vertx-starter-ui",
   "version": "1.0.0",
   "description": "UI for the Kafka Java Vertx starter application",
-  "main": "src/index.js",
+  "author": "",
+  "license": "Apache-2.0",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ibm-messaging/kafka-java-vertx-starter.git"
+  },
   "scripts": {
-    "test": "test",
-    "build": "ls"
+    "start": "webpack-dev-server --mode development",
+    "build": "webpack --mode production",
+    "test": "jest",
+    "storybook": "start-storybook"
   },
-  "author": "",
-  "license": "Apache-2.0"
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "dependencies": {
```

Comment:
Can all dependancies which make up the UI be made a dependancy, and all assisting development become a dev dependancy? Ie, can the testing libraries go to dev?